src/app/mytools/my-tool/my-tool.component.ts

```diff
@@ -150,6 +150,7 @@ export class MyToolComponent extends MyEntry implements OnInit {
     this.router.events
       .pipe(
         filter((event) => event instanceof NavigationEnd),
+        map((event) => event as NavigationEnd),
```

Comment:
Compiler complained without adding this. I guess Angular 16 compiler is stricter about some things.